eks_infra/env_setup.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#/ Usage: ./env_setup.sh
+
+set -ex
+
+function install_kubectl_client() {
+    curl --silent --location https://amazon-eks.s3-us-west-2.amazonaws.com/1.20.4/2021-04-12/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl
```

Comment:
can we get this `1.20.4` from the json file? if I update the params, for instance, I still need to update these values separately

Reply:
Its not just the `1.20.4` value, the entire URL needs to be replaced with a new value for every version

Reply:
Is it possible for us to set all the versions and links associated with those versions in one place?

Reply:
Moved the URL/versions at the top of the file.